sources/subs/Notification.subs.php

```diff
@@ -1075,6 +1076,11 @@ function saveUserNotificationsPreferences($member, $notification_data)
 	$inserts = array();
 	foreach ($notification_data as $type => $level)
```

Comment:
Took me a while but here I am.
Yes, this is indeed true, technically it would even be _n_ choices times _m_ notifications, considering addons can implement different choices and different types of notifications.
How would you feel about moving to MySQL 5.7.8 to use the JSON type? :angel: 
Not sure if worth at that point, also considering the fact that writing code to support both MySQL/Mariadb and PostgreSQL would be a bit annoying... :-\

Reply:
Had not thought about the JSON db support !

Honestly we probably should support that as the serialized stuff really is, yea.  Looks like PostgreSQL 9.2 supports JSON, not sure if it would need a translator or not?   Upping the min db required is not a problem IMO  as long as it supported by 10.3.25-MariaDB since that is what the site is running and I don't want to bug TE again just yet 👅